docs/competition/livestream/scenes.md

```diff
@@ -0,0 +1,50 @@
+# Scenes
+
+This page documents a few of the core scenes which are used for livestreams. Not all of them are applicable to all events, but most are.
+
+The various assets useful for creating these scenes can be found [here](https://drive.google.com/drive/folders/1pr1vKpnonxFoO8O5chsH5DF7Vsd0Tfih?usp=sharing).
```

Comment:
I think these need a more permanent home. That drive link was kind of just a dump of everything I had.

Reply:
This isn't your drive link. I made a new one in the shared competition drive, so this is probably as good of a home as any?

Reply:
I think this is fine for now. Longer term it would be great to great to have them in a form which eases contributions? It's not clear to me for example how one would make or re-create one of the images there. Happy for this to be picked up separately though.